sentry-ruby/lib/sentry/transaction.rb

```diff
@@ -94,6 +100,20 @@ def deep_dup
       copy
     end
 
+    # Sets a custom measurement on the transaction.
+    # @param name [String] name of the measurement
+    # @param value [Float] value of the measurement
+    # @param unit [String] unit of the measurement
+    # @return [void]
+    def set_measurement(name, value, unit = "")
```

Comment:
@sl0thentr0py It looks like we'll allow users to pass custom unit? If so, I'm not sure it'd be helpful to store a list of default units as we won't be able to hint users about them anyway. 

It'd be different if we decide to enforce them in Ruby SDK though. lmk what you think.

Reply:
we support arbitrary units so we decided there would be no validation in the SDKs.

Reply:
Will official unit names like `bit` get UI highlight? Or `bit` will be displayed the same as `foobarbaz`. 
Asking because I wonder if we need to maintain the list of units as documentation if name variations like `bits` or `Bit` will make no difference.

Reply:
We will eventually have a common docs page for supported units which all SDKs can link to.

Reply:
That's great 👍